news/2023/2023-09-21-osu-world-cup-2023-registrations-now-open.md

```diff
@@ -1,47 +1,47 @@
-| Grand Finals | Dec 02 & Dec 03 |
-
-## Prizes
-
-The osu! World Cup 2023 offeres a $5,000 cash prize pool and limited-edition merch!
-
-- **1st:** 48% of the prize pool, single-run merch, profile badge, **osu! Champion** user title for one year
-- **2nd:** 32% of the prize pool, single-run merch, profile badge
-- **3rd:** 20% of the prize pool, single-run merch, profile badge
-
-- **[Click here!](https://osu.ppy.sh/community/tournaments/41)**
-- Teams will be formed by each country's captain *after* the registration phase has concluded.
-- **The deadline for registering is September 5, 23:59 UTC. Make sure to submit yours before then!**
-
-*Notice: you may only participate if your position in the osu! global ranking is #5,000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months.*
-
-If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
-
-*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of time to avoid any last-minute headaches. If you run into any issues, send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).*
-
-—LeoFLT
+---
+layout: post
+title: "osu! World Cup 2023: Registrations Now Open!"
+date: 2023-09-21 00:00:00 +0000
+---
+
+The definitive country-based circle clicking competition is back. The osu! World Cup 2023 is now open for registrations!
+
+![](/wiki/Tournaments/OWC/2023/img/owc2023-banner.jpg)
+
+After smashing keys to the beat, drumming to the beat, catching fruit to the beat, and smashing even more keys, it's time to click the circles to the beat, as the osu! World Cup comes back for its 14th instalment! 
+
+For more details, refer to the [osu! World Cup 2023 wiki article](/wiki/Tournaments/OWC/2023). You can discuss this event in the [discussion thread](https://osu.ppy.sh/community/forums/topics/1823846) or at the public [osu! community server](https://discord.gg/0Vxo9AsejDkGlk3H).
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | Sep 21 – Oct 05 |
+| Qualifier showcase | Sep 15 (14:00 UTC) |
+| Qualifier stage | Sep 21 & Sep 22 |
+| Round of 32 | Sep 28 & Sep 29 |
+| Round of 16 | Nov 04 & Nov 05 |
+| Quarterfinals | Nov 11 & Nov 12 |
+| Semiinals | Nov 18 & Nov 19 |
+| Finals | Nov 25 & Nov 26 |
+| Grand Finals | Dec 02 & Dec 03 |
+
+## Prizes
+
+The osu! World Cup 2023 offeres a $5,000 cash prize pool and limited-edition merch!
+
+- **1st:** 48% of the prize pool, single-run merch, profile badge, **osu! Champion** user title for one year
+- **2nd:** 32% of the prize pool, single-run merch, profile badge
+- **3rd:** 20% of the prize pool, single-run merch, profile badge
+
+- **[Click here!](https://osu.ppy.sh/community/tournaments/41)**
```

Comment:
I'm pretty sure you're going to want to fix this. Having to unrelated bulleted lists next to each other is pretty ugly.

Reply:
replaced by a regular section